File: python練習/Rectangle_Encapsulation/Rectangle.py
class Rectangle:
    def __init__(self, length, width):
        self.__length = length
        self.__width = width
        
    def setLength(self, leng):
        self.__length = leng
    
    def setWidth(self, wid):
        self.__width = wid
        
    def getLength(self):
        return self.__length
    
    def getWidth(self):
        return self.__width
    
    def getArea(self):
        return self.getLength() * self.getWidth()
    
    def getPerimeter(self):
        total = int(self.getWidth()) * 2 + int(self.getLength()) * 2
        return total

File: 9章/9-2/Rectangle.cpp
#include "Rectangle(2).h"

Rectangle::Rectangle()
{
	width = 1;
	height = 1;
}

Rectangle::Rectangle(double newHeight, double newWidth)
{
	height = newHeight;
	width = newWidth;
}

double Rectangle::getArea()
{
	return height * width;
}

double Rectangle::getPerimeter()
{
	return (height * 2) + (width * 2);
}
